cpp/ql/test/library-tests/ir/ir/aliased_ir.expected

```diff
@@ -13269,6 +13453,852 @@ ir.cpp:
+# 2275|     m2275_2(unknown)        = AliasedDefinition               : 
+# 2275|     m2275_3(unknown)        = InitializeNonLocal              : 
+# 2275|     m2275_4(unknown)        = Chi                             : total:m2275_2, partial:m2275_3
+# 2275|     r2275_5(glval<bool>)    = VariableAddress[b]              : 
+# 2275|     m2275_6(bool)           = InitializeParameter[b]          : &:r2275_5
+# 2276|     r2276_1(glval<String>)  = VariableAddress[s]              : 
+# 2276|     m2276_2(String)         = Uninitialized[s]                : &:r2276_1
+# 2276|     r2276_3(glval<unknown>) = FunctionAddress[String]         : 
+# 2276|     v2276_4(void)           = Call[String]                    : func:r2276_3, this:r2276_1
+# 2276|     m2276_5(unknown)        = ^CallSideEffect                 : ~m2275_4
+# 2276|     m2276_6(unknown)        = Chi                             : total:m2275_4, partial:m2276_5
+# 2276|     m2276_7(String)         = ^IndirectMayWriteSideEffect[-1] : &:r2276_1
+# 2276|     m2276_8(String)         = Chi                             : total:m2276_2, partial:m2276_7
+# 2277|     r2277_1(glval<bool>)    = VariableAddress[b]              : 
+# 2277|     r2277_2(bool)           = Load[b]                         : &:r2277_1, m2275_6
+# 2277|     v2277_3(void)           = ConditionalBranch               : r2277_2
+#-----|   False -> Block 4
+#-----|   True -> Block 3
+
+# 2275|   Block 1
+# 2275|     m2275_7(unknown) = Phi          : from 2:~m2284_21, from 2:~m2284_5, from 5:~m2284_21, from 5:~m2284_5, from 3:~m2284_13, from 2:~m2284_21, from 2:~m2284_5, from 5:~m2284_21, from 5:~m2284_5, from 6:~m2284_30
+# 2275|     v2275_8(void)    = ReturnVoid   : 
+# 2275|     v2275_9(void)    = AliasedUse   : ~m2275_7
+# 2275|     v2275_10(void)   = ExitFunction : 
+
+# 2278|   Block 3
+# 2278|     v2278_1(void)            = NoOp                            : 
+# 2284|     r2284_9(glval<String>)   = VariableAddress[s]              : 
+# 2284|     r2284_10(glval<unknown>) = FunctionAddress[~String]        : 
+# 2284|     v2284_11(void)           = Call[~String]                   : func:r2284_10, this:r2284_9
+# 2284|     m2284_12(unknown)        = ^CallSideEffect                 : ~m2276_6
+# 2284|     m2284_13(unknown)        = Chi                             : total:m2276_6, partial:m2284_12
+# 2284|     v2284_14(void)           = ^IndirectReadSideEffect[-1]     : &:r2284_9, m2276_8
+# 2284|     m2284_15(String)         = ^IndirectMayWriteSideEffect[-1] : &:r2284_9
+# 2284|     m2284_16(String)         = Chi                             : total:m2276_8, partial:m2284_15
+#-----|   Goto -> Block 1
+
+# 2280|   Block 4
+# 2280|     r2280_1(glval<bool>) = VariableAddress[b] : 
+# 2280|     r2280_2(bool)        = Load[b]            : &:r2280_1, m2275_6
+# 2280|     v2280_3(void)        = ConditionalBranch  : r2280_2
+#-----|   False -> Block 6
+#-----|   True -> Block 5
+#-----|   True -> Block 5
```

Comment:
This duplication looks wrong, but I can reproduce this on `main` already. So no need to investigate that as part of this PR.

Reply:
Fixed in dd97584eff2c65edcf6579060efd3ec7043eddbf